### DISCUSSION TITLE
SWX LIVE (KHQ-DT2)

### DISCUSSION THREAD

**Discussion by R4IN2007** [iptv-org/community#358] [Show and tell]
### Channel ID

KHQDT2.us

### Stream URL

https://www.khq.com/sports/swx_live/

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by Alstruit** [comment]
Correction: Still pending.

**Comment by BellezaEmporium** [comment]
https://fuel-streaming-prod01.fuelmedia.io/v1/sep/9a4ed4e6-bc16-4154-95a1-6fa49db8024b/600K360AWS/SM/content.m3u8 (from website, USA IP)

It is **not** a live stream per-se, but it's a reconstitution of something.

**Comment by AntiPontifex** [comment]
Stream dead: please close.

**Comment by freearhey** [comment]
https://fuel-streaming-prod01.fuelmedia.io/v1/sep/9a4ed4e6-bc16-4154-95a1-6fa49db8024b/600K360AWS/SM/content.m3u8 (offline)
